sdks/java/io/kudu/build.gradle

```diff
@@ -21,6 +21,8 @@ applyJavaNature(
   classesTriggerCheckerBugs: [
     'KuduTestUtils': 'TODO: file a bug report',
     'KuduIOIT': 'TODO: file a bug report',
+    'KuduIO': 'TODO: file a bug report', // only crashes under Java 11
```

Comment:
LGTM in case someone wants to build this branch with Java 11. But the official release images should still be built with Java 8, so we should proceed with caution.

Reply:
It's possibly a different misconfiguration of my system. However, I'd like to get to the end of this particular release candidate anyway. There are still a few blocking issues with the candidate, but I'd like to enable folks to find others.